insightvm_connector.py

```diff
@@ -66,193 +69,178 @@ def finalize(self):
 
         return phantom.APP_SUCCESS
 
-    def _login(self, action_result):
-
-        config = self.get_config()
-
-        endpoint = 'LoginRequest'
-
-        params = {'user-id': config[phantom.APP_JSON_USERNAME], 'password': config[phantom.APP_JSON_PASSWORD]}
-
-        ret_val, resp_data = self._make_soap_call(action_result, endpoint, params)
-
-        if not ret_val:
-            return ret_val
-
-        self._session_id = dict(resp_data).get('LoginResponse', {}).get('@session-id')
-
-        if not self._session_id:
-            return action_result.set_status(phantom.APP_ERROR, consts.INSIGHTVM_ERR_NO_SESSION_ID)
-
-        return phantom.APP_SUCCESS
-
-    def _process_request_exception(self, exception):
-
-        addition = ''
-        e_str = str(exception)
-        if 'Max retries exceeded' in e_str:
-            addition = consts.INSIGHTVM_ERR_BAD_IP
-        elif 'bad handshake' in e_str or '_ssl.c:504' in e_str:
-            addition = consts.INSIGHTVM_ERR_BAD_CERT
-
-        if not addition:
-            return consts.INSIGHTVM_ERR_SERVER_CONNECTION.format(exception)
-
-        self.save_progress(consts.INSIGHTVM_ERR_SERVER_CONNECTION.format(exception))
-        return addition
-
     def _process_empty_reponse(self, response, action_result):
 
         if response.status_code == 200:
             return RetVal(phantom.APP_SUCCESS, {})
 
-        return RetVal(action_result.set_status(phantom.APP_ERROR, "Empty response and no information in the header"), None)
+        return RetVal(
+            action_result.set_status(phantom.APP_ERROR, "Empty response and no information in the header"),
+            None
+        )
 
     def _process_html_response(self, response, action_result):
 
-        # An html response, is bound to be an error
+        # An html response, treat it like an error
         status_code = response.status_code
 
         try:
             soup = BeautifulSoup(response.text, "html.parser")
```

Comment:
Add below code to remove unnecessary things from the HTML message.

```
# Remove the script, style, footer and navigation part from the HTML message
	for element in soup(["script", "style", "footer", "nav"]):
		element.extract()
```

Reply:
updated